ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/port_channel_interfaces.py

```diff
@@ -100,11 +105,12 @@ def port_channel_interfaces(self) -> dict | None:
                                 interface["rt"] = generate_route_target(short_esi)
                                 if port_channel_mode == "active":
                                     interface["lacp_id"] = generate_lacp_id(short_esi)
-                        port_channel_interfaces[interface_name] = interface
 
-            subif_parent_interfaces = {key: value for key, value in subif_parent_interfaces.items() if key not in port_channel_interfaces}
-            if subif_parent_interfaces:
-                port_channel_interfaces.update(subif_parent_interfaces)
+                        port_channel_interfaces.append(interface)
+
+            port_channel_interfaces.extend(
+                [subif_parent_interface for subif_parent_interface in subif_parent_interfaces if subif_parent_interface not in port_channel_interfaces]
+            )
```

Comment:
do you need the `[]` or does this work:
```suggestion
            port_channel_interfaces.extend(
                subif_parent_interface for subif_parent_interface in subif_parent_interfaces if subif_parent_interface not in port_channel_interfaces
            )
```


looks like it works fine:
```
>> test = []
>>> test.extend(i for i in range(1,10))
>>> test
[1, 2, 3, 4, 5, 6, 7, 8, 9]
>>>
```